packages/ia-components/sandbox/track-lists/track-list.jsx

```diff
@@ -23,7 +23,7 @@ const parseTrackTitle = ({
   isAlbum = false
 }) => {
   if (isAlbum) { return 'Full album'; }
-  const albumCreatorString = albumCreator.join('; ');
+  const albumCreatorString = albumCreator.join ? albumCreator.join('; ') : albumCreator;
```

Comment:
**Suggestion**: Instead of relying on the only object type to have a `join` method be an array, test it with `Array.isArray`

Reply:
thanks for the catch, forgot to switch it out to that